crates/service_utils/src/service/types.rs

```diff
@@ -208,3 +209,24 @@ impl FromRequest for DbConnection {
         ready(result)
     }
 }
+
+pub struct CustomHeaders {
+    pub config_tags: Option<String>,
```

Comment:
I have to perform split on string , so the logic is extract any custom headers as option<string> , and then in the next step we can add parsing and validation logic, here we should not add because these operation is individual based on headers

Reply:
:+1: